src/coreComponents/constitutive/unitTests/testLinearElasticAnisotropic.cpp

```diff
@@ -40,7 +40,7 @@ TEST( LinearElasticAnisotropicTests, testAllocation )
 //  arrayView1d<LinearElasticAnisotropic::StiffnessTensor const> const &
 //  stiffness = cm.stiffness() ;
 
-  arrayView3d< real64 const, solid::STRESS_USD > const & stress = cm.getStress().toViewConst();
```

Comment:
So the conversion is automatic now, or was it redundant to begin with?

Reply:
The conversion from `ArrayView< T > &&` to `ArrayView< T const >` is now automatic.